manager/src/agent.rs

```diff
@@ -2,12 +2,34 @@ use near_contract_standards::storage_management::StorageManagement;
 
 use crate::*;
 
+#[derive(BorshDeserialize, BorshSerialize, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(crate = "near_sdk::serde")]
+pub enum AgentStatus {
+    // Default for any new agent, if tasks ratio allows
+    Active,
+
+    // Default for any new agent, until more tasks come online
+    Pending,
+}
+
 #[derive(BorshDeserialize, BorshSerialize, Debug, Serialize, Deserialize, PartialEq)]
 #[serde(crate = "near_sdk::serde")]
 pub struct Agent {
+    pub status: AgentStatus,
```

Comment:
I don't think we usually have to have `pub` before the fields here, just on the `pub struct…`

Reply:
With the file re-org, its now required for rust server